### DISCUSSION TITLE
FreeBSD

### DISCUSSION THREAD

**Discussion by sonicaghi** [Kong/kong#7196] [Ideas and feature requests]
Create a package distribution compatible with FreeBSD.

**Comment by subnetmarco** [comment]
It would be nice if fpm could support this (jordansissel/fpm/issues/62)

**Comment by kylegato** [comment]
Would love to have this!

**Comment by pshanmu** [comment]
Hi,

Does kong work on Freebsd 10.x ? is it supported.

i am getting this error.

luarocks install kong 0.9.2-0
Installing https://luarocks.org/kong-0.9.2-0.rockspec...
Using https://luarocks.org/kong-0.9.2-0.rockspec... switching to 'build' mode

Error: This rockspec for kong does not support unix, bsd, freebsd platforms.

**Comment by subnetmarco** [comment]
Currently not supported.

**Comment by pshanmu** [comment]
Thanks Marco for the prompt response.
- Shanmu

**Reply by sppg** [reply]
Hello, just passed by and found the discussion.
We also need kong on FreeBSD, does anyone know what was done and what was blocking ? so we'll continue where you stopped to not loose your work and trials.
Best regards

**Comment by tobiastom** [comment]
Is there any plan to support it in the future? I'd also like to use it on FreeBSD.

**Comment by urosgruber** [comment]
It's been a long time and I'm not sure if anything was done on FreeBSD support?

**Comment by jhfoo** [comment]
Wondering if the fundamental blocker is jail support?
